AutoPartsShop.API/Models/CartItem.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace AutoPartsShop.API.Models
+{
+    public class CartItem
+    {
+        public int Id { get; set; }
+        public int CartId { get; set; }
```

Comment:
I think it would be easier for you to work with these models if the `CartId` from `CartItem` class will be removed and a list of `CartItems` will be added in the `Cart` class.